macos/RNCPicker.m

```diff
@@ -20,11 +20,11 @@ - (instancetype)initWithFrame:(CGRect)frame
 {
     if ((self = [super initWithFrame:frame pullsDown:false])) {
         _color = [NSColor blackColor];
-        _font  = [NSFont systemFontOfSize:21];
+        _customFont  = [NSFont systemFontOfSize:14];
```

Comment:
Curious why renaming `_font` to `_customFont` fixes it.

Reply:
font is an inherited property of the pop up